pkg/ovs/openflow/interfaces.go

```diff
@@ -21,6 +21,7 @@ import (
 
 type protocol = string
 type TableIDType uint8
+type GroupIDType = uint32
```

Comment:
I forget the difference between type definition with "=" and without "=" again, but can we unify the style? I believe the difference doesn't apply to TableIDType and GroupIDType whatever it is?

Reply:
Agree, because TypeAlias could reduce unneeded type casting I would like to unify these statement to this form. `binding.Range` implemented its own methods, thus it must be a type but not alias.
FYI. https://go.googlesource.com/proposal/+/master/design/18130-type-alias.md

Reply:
I believe we should avoid type aliases (with "=") altogether. If I recall correctly, type aliases were introduced specifically to solve some refactoring issues?

Reply:
You're right, I searched for some best practices.
> Type aliases are not meant for everyday use. They were introduced to support gradual code repair while moving a type between packages during large-scale refactoring. Codebase Refactoring (with help from Go) covers this in detail.
I would make them as type instead alias.